src/main/java/fr/inria/coming/repairability/RepairabilityAnalyzer.java

```diff
@@ -81,4 +92,19 @@ public AnalysisResult analyze(IRevision input, RevisionResult previousResults) {
 
         return finalResult;
     }
+
+    private boolean coversTheWholeDiff(ChangePatternInstance instancePattern, Diff diff) {
+        for(Operation diffOperation : diff.getAllOperations()){
```

Comment:
per the discussion on Github and today in person, we change this to getRootOperations